Resources/OR/Login_OR.resource

```diff
@@ -0,0 +1,15 @@
+*** Comments ***
+Object Repository for Login Screen
+
+*** Variables ***
+${titleLoginType}           //h1[contains(.,"Log in with")]
```

Comment:
Can we use aid? (automation ids)

Reply:
@ikanias I am considering id's as the first choice of object locators, For the object doesnt have id attribute - xpath used